### ISSUE TITLE
Double-clicking, w or q select only works in a per-player view for spectators

### ISSUE THREAD

**Issue by netnazgul** [OpenRA/OpenRA#16044]
WRT @dragunoff 's quality of life improvements.
I think it actually worked for "All players" view as well, and it seems more useful that way too. At the moment double-click and "w" don't select any additional actors, and "q" drops selection.

**Comment by abcdefg30** [comment]
Actually, this looks like a duplicate of #9492. (?)

**Comment by dragunoff** [comment]
My PR (#15813) fixes just that but it has not been merged yet. This issue is not related to the playtest, it exists in release as well.

**Comment by netnazgul** [comment]
@abcdefg30 I thought that #15813 was merged in in the hotfix and/or broken afterwards.

**Comment by pchote** [comment]
Can anyone confirm whether this is a duplicate of #9492 and/or whether this is a regression from the behaviour in release-20181215?

**Comment by pchote** [comment]
Confirmed that the behaviour on bleed/prep matches release-20181215 so bumping to Next + 1 to unblock the next playtest.
